CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog for Bitmark SDK for Java
+
+## 2.1.0
```

Comment:
I thought following things are mentioned is existing in prior 2.1.0, so i think we should not put existing features here, only new features should stand here.